### PR TITLE
Tests!

### PR DESCRIPTION
**Pull request by CloudNiner**
Tests for all the services/controllers I added yesterday (I think).

### CODE REVIEW

**Comment by CloudNiner** [comment]
@moradology @kshepard might want to take a look at this as well. There are one or two slightly  more complicated examples.

**Comment by flibbertigibbet** [comment]
I wonder why it's always the lodash npm lockfile that Travis seems to trip up on.

File: src/nih_wayfinding/app/scripts/mapping/map-control-service.js [hunk 1]
```diff
@@ -21,6 +21,9 @@
             if (!lonLat) {
                 return null;
             }
+            if (!lonLat.split) {
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/mapping/map-control-service.js]: Maybe just combine this with the above `if` and `or` the conditions together.

File: src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js [hunk 2]
```diff
@@ -49,10 +49,15 @@
                     var currentPosition = [position.coords.longitude, position.coords.latitude];
                     if (!destination) {
                         destination = currentPosition;
+                    }
+                    if (!origin) {
+                        origin = currentPosition;
+                    }
+                    if (_.isEqual(origin, destination)) {
                         directionsOptions.walkTimeMins = 30;
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js]: Might want to define this in a constant up top.

File: src/nih_wayfinding/test/spec/mapping/map-service.spec.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        expect(Map.center).toBeDefined();
+        expect(Map.center.lat).toBeDefined();
+        expect(Map.center.lng).toBeDefined();
+        expect(Map.center.zoom).toBeDefined();
+        expect(Map.layers).toBeDefined();
+    });
+
+    it('should ensure that geojson is not defined', function () {
+        // if geojson is initialized to an empty object, leaflet-directive won't pick up
+        // the changes when updated
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/test/spec/mapping/map-service.spec.js:26]: Good to know.

File: src/nih_wayfinding/test/spec/views/locations/locations-controller.spec.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  }));
+
+  it('should initialize findAddressExpanded to false', function () {
+    expect(LocationsController.findAddressExpanded).toEqual(false);
+  });
+  it('should initialize searchText to empty string', function () {
+    expect(LocationsController.searchText).toEqual('');
+  });
+
+  it('should expect search() to trigger a state change to routing', function () {
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/test/spec/views/locations/locations-controller.spec.js:48]: Neat!

**Comment by flibbertigibbet** [comment]
+1, pending Travis pass.

**Comment by kshepard** [comment]
Excellent, LGTM.

**Pull request closed without merging**